docs/training-apis/api-core/metrics.txt

```diff
@@ -0,0 +1,56 @@
+.. _core-metrics:
+
+################
+ Report Metrics
+################
+
+The Core API makes it easy to report training and validation metrics to the master during training
+with only a few new lines of code.
+
+For this example, create a new training script called ``1_metrics.py`` by copying the ``0_start.py``
+script from :ref:`core-getting-started`.
+
+Begin by importing import the ``determined`` module:
+
+.. literalinclude:: ../../../examples/tutorials/core_api/1_metrics.py
+   :language: python
+   :start-after: NEW: import determined
+   :end-before: def main
+
+In your ``if __name__ == "__main__"`` block, wrap the entire execution of ``main()`` within the
+scope of :meth:`determined.core.init`, which prepares resources for training and clean them up
```

Comment:
Should be "clean**s** them up", although, something less folksy would be even better.

Reply:
prepares resources for training and then tears them down?

Not sure how else to phrase it:
prepare / clean up
set up / tear down

Reply:
"...releases the resource." Maybe, as something more specific than "cleans up"?